src/RawTransaction.php

```diff
@@ -934,9 +932,6 @@ public static function validate_signed_transaction($raw_tx, $json_string, $magic
         $magic_p2sh_byte = BitcoinLib::magicP2SHByte($magic_p2sh_byte);
 
         $decode = self::decode($raw_tx, $magic_byte, $magic_p2sh_byte);
-        if ($decode == false) {
-            return false;
-        }
```

Comment:
decode no longer returns false on failure but throws so no longer need this check